### PR TITLE
feat(molecule/badgeCounter): add Large size and a custom prop to MoleculeBadgeConter

### PR DESCRIPTION
**Pull request by AitorRodriguez990**
## [Molecule]/[BadgeCounter]
**TASK**: [SCMI-47974](https://jira.scmspain.com/browse/SCMI-47974)
**ISSUE**: [#1256](https://github.com/SUI-Components/sui-components/issues/1256)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description, Motivation and Context

We need to iterate this component to make it bigger and customizable.

Our design will be:

image

### Screenshots - Animations

In progress.

image

### CODE REVIEW

**Comment by turolopezsanabria** [comment]
cool!
Can you add please a screenshot for this change?

**Comment by AitorRodriguez990** [comment]
> cool!
> Can you add please a screenshot for this change?

Done 😉

**Review by turolopezsanabria — APPROVED** [review]
👏  merci

**Review by adriadoz — APPROVED** [review]

**Review by danivm — APPROVED** [review]

File: components/molecule/badgeCounter/src/index.js [hunk 1]
```diff
@@ -54,16 +68,36 @@ const MoleculeBadgeCounter = ({children, label = '', size, variant}) => {
...
+  return CLASS_LARGE_THREE_CHARS
+}
+
 MoleculeBadgeCounter.displayName = 'MoleculeBadgeCounter'
 
 MoleculeBadgeCounter.propTypes = {
   /** children */
   children: PropTypes.node,
 
+  /** Custom */
```
Comment (andresin87) [inline components/molecule/badgeCounter/src/index.js]: this description is not enough

Comment (AitorRodriguez990) [reply components/molecule/badgeCounter/src/index.js]: Done!

File: components/molecule/badgeCounter/src/index.scss [hunk 2]
```diff
@@ -2,12 +2,16 @@
 
 $bg-molecule-badge-counter: #ff0057 !default;
 $c-molecule-badge-counter: $c-white !default;
-$br-molecule-badge-counter: $bdw-xl !default;
+$br-molecule-badge-counter: 16px !default;
```
Comment (andresin87) [inline components/molecule/badgeCounter/src/index.scss]: $**bdrs**-molecule-badge-counter: calc(2 * $bdrs-xl) !default;
[sui-theme](https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss)

**Review by javiauso — APPROVED** [review]

File: components/molecule/badgeCounter/src/index.scss (hunk 2 above)
Comment (turolopezsanabria) [reply components/molecule/badgeCounter/src/index.scss]: we can leave it as @andresin87 suggests here, so we don't "mareamos más la perdiz", but this adjustment is assuming all themes have a defined value for $bdrs-xl of 8px (which is not correct)

Comment (andresin87) [reply components/molecule/badgeCounter/src/index.scss]: This can become a major version upgrade. Do u agree?
`$bdrs-`

Comment (turolopezsanabria) [reply components/molecule/badgeCounter/src/index.scss]: yeapi, I agree we can put the `calc(2 * $bdrs-xl) !default;` in a major, after checking the side effects

Comment (AitorRodriguez990) [reply components/molecule/badgeCounter/src/index.scss]: We can do a major for `$bg` too, because it must be `$bgc`. But I think it's too much.

Comment (andresin87) [reply components/molecule/badgeCounter/src/index.scss]: `br-molecule-badge-counter`--> `bdrs-molecule-badge-counter`is the major i think. cuz it refers to border-radius and naming change supose a major

Comment (andresin87) [reply components/molecule/badgeCounter/src/index.scss]: > we can leave it as @andresin87 suggests here, so we don't "mareamos más la perdiz", but this adjustment is assuming all themes have a defined value for $bdrs-xl of 8px (which is not correct)

maybe `16px`is ok, but the token naming is not

**Review by andresin87 — APPROVED** [review]

**Pull request merged**